examples/md/copymd.ts

```diff
@@ -0,0 +1,15 @@
+import { copyHTML } from "../../copyhtml/copyhtml.ts";
+
+export const copyMD = copyHTML.bind(null, {
+  transform(elements: NodeListOf<Element>) {
+    const html = elements[0]?.outerHTML;
+    if (!html) {
+      return;
+    }
+
+    const converter = new window.showdown.Converter();
```

Comment:
Instead of assuming the `showdown` module is attached to the `window` object, let's import `showdown` to bundle the code as part of the output bookmarklet.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/194339961caddbcd7a91c8428ace324a2cd5f209>.